### DISCUSSION TITLE
iped-hashes.db available for use

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#2167] [Show and tell]
We created a hash base iped-hashes.db containing the NIST bases which is around 40GB

RDS_2024.03.1_android
RDS_2024.03.1_ios
RDS_2024.03.1_legacy
RDS_2024.03.1_modern

To use, simply configure it in localconfig.txt where you saved it. Example
hashesDB = iped-hashes/iped-hashes.db

You can download it from the link below, which is about 16GB because I compressed it.
User  Ramiro Marín made the space available for around 30 days. If anyone has space available to leave it, it would be interesting.

https://e.pcloud.link/publink/show?code=kZQhI9Z7K6604TAIfQnuEi472cxoQMsRvAX

@lfcnassif  remembered that you need to turn on enableHashDBLookup in IPEDConfig.txt and excludeKnown in conf/HashDBLookupConfig.txt
This way you will ignore system files, not being included in the case.
Remember to put this base on an SSD

**Comment by joasource** [comment]
> We created a hash base iped-hashes.db containing the NIST bases which is around 40GB
> 
> RDS_2024.03.1_android RDS_2024.03.1_ios RDS_2024.03.1_legacy RDS_2024.03.1_modern
> 
> To use, simply configure it in localconfig.txt where you saved it. Example hashesDB = iped-hashes/iped-hashes.db
> 
> You can download it from the link below, which is about 16GB because I compressed it. User Ramiro Marín made the space available for around 30 days. If anyone has space available to leave it, it would be interesting.
> 
> https://e.pcloud.link/publink/show?code=kZQhI9Z7K6604TAIfQnuEi472cxoQMsRvAX
> 
> @lfcnassif remembered that you need to turn on enableHashDBLookup in IPEDConfig.txt and excludeKnown in conf/HashDBLookupConfig.txt This way you will ignore system files, not being included in the case. Remember to put this base on an SSD

The link has expired. How can I obtain the updated files?

**Comment by antomiccoli** [comment]
Hi PAULOBREIN whre I can find the hashes database?
Thank's

**Reply by paulobreim** [reply]
Do you want the final database, iped-hashes.db?
It's about 40GB, if you have a place to upload it I can send it.
Next week I can put it on a server for download.

But you can get RDS_2024.03.1_android RDS_2024.03.1_ios RDS_2024.03.1_legacy and generate another one like it

**Comment by antomiccoli** [comment]
Great
Thank's

Il Mer 24 Lug 2024, 23:45 Paulo Cesar Breim (PCB) ***@***.***>
ha scritto:

**Reply by paulobreim** [reply]
I compressed the file, which reduced it from 48GB to 16GB.
You can download it from this link:
https://mobile.canalvip.com.br:44443/iped-hashes.7z

**Comment by antomiccoli** [comment]
Thank's
Have a nice WE

Il giorno sab 27 lug 2024 alle ore 02:43 Paulo Cesar Breim (PCB) <
***@***.***> ha scritto:

**Comment by apcjoaofilho** [comment]
> We created a hash base iped-hashes.db containing the NIST bases which is around 40GB
> 
> RDS_2024.03.1_android RDS_2024.03.1_ios RDS_2024.03.1_legacy RDS_2024.03.1_modern
> 
> To use, simply configure it in localconfig.txt where you saved it. Example hashesDB = iped-hashes/iped-hashes.db
> 
> You can download it from the link below, which is about 16GB because I compressed it. User Ramiro Marín made the space available for around 30 days. If anyone has space available to leave it, it would be interesting.
> 
> https://e.pcloud.link/publink/show?code=kZQhI9Z7K6604TAIfQnuEi472cxoQMsRvAX
> 
> @lfcnassif remembered that you need to turn on enableHashDBLookup in IPEDConfig.txt and excludeKnown in conf/HashDBLookupConfig.txt This way you will ignore system files, not being included in the case. Remember to put this base on an SSD

The link is expired. Please send again.

**Comment by joasource** [comment]
> We created a hash base iped-hashes.db containing the NIST bases which is around 40GB
> 
> RDS_2024.03.1_android RDS_2024.03.1_ios RDS_2024.03.1_legacy RDS_2024.03.1_modern
> 
> To use, simply configure it in localconfig.txt where you saved it. Example hashesDB = iped-hashes/iped-hashes.db
> 
> You can download it from the link below, which is about 16GB because I compressed it. User Ramiro Marín made the space available for around 30 days. If anyone has space available to leave it, it would be interesting.
> 
> https://e.pcloud.link/publink/show?code=kZQhI9Z7K6604TAIfQnuEi472cxoQMsRvAX
> 
> @lfcnassif remembered that you need to turn on enableHashDBLookup in IPEDConfig.txt and excludeKnown in conf/HashDBLookupConfig.txt This way you will ignore system files, not being included in the case. Remember to put this base on an SSD

Greetings. I have a lot of extra space in my Google Drive. I can share a link.

**Reply by SuelenSaraM** [reply]
@joasource, could you share a Google Drive link with me too, please?

**Reply by paulobreim** [reply]
Yes, here is:

https://mobile.canalvip.com.br/iped-hashes.db


ps: canalvip.com.br was the first .com.br domain to be registered in Brazil ;-)

**Comment by antomiccoli** [comment]
I'M Sorry, for what?

Il giorno mar 22 ott 2024 alle ore 15:10 SuelenSaraM <
***@***.***> ha scritto:
